andela/goodfellas-ah-backend
# Feature/16061810/create user profile

### What does this PR do?
This PR enables the users to create and update their profile  
Also enables users to view any profile

### Description of Task to be completed?
- Create an initial user profile on signup
- View user profile
- Update user profile
-Upload profile image
- Validate inputs/fields

### How should this PR be manually tested
- Clone the repo
- run `npm install`
- signup by sending a `POST` request to  `api/auth/signup` route an initial profile will be created
- view profile by sending a `GET` request `api/profile/user/id` 
- update profile and upload image by sending a `PUT` request to `api/profile/user`
- view profile to see the updated profile details

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
#160618103

### Screenshots (if appropriate)
N/A

### Questions:
None
